### ISSUE TITLE
Audit & document use of  Bold text on headings 

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/va.gov-team#19573]
## Issue Description
During the 2/5 DSC meeting, Trevor had raised the topic of bold text turning into HTML on scheduling cards. The DST has been tasked with doing an audit of this pattern and draft guidelines. 

Link to #19301 has more information. 
Bold text on headings must be consistently marked up as headings
https://github.com/department-of-veterans-affairs/va.gov-team/issues/19301

There is also a Slack conversation about this that Ryan shared. 
 https://dsva.slack.com/archives/C0FQSS30V/p1611085952085600

---
## Tasks
- [x] Audit use of pattern 

## Acceptance Criteria
- [x] Audit of current usage of bold text vs H4 usage for headings 

---

**Comment by caw310** [comment]
This item originated from a discussion about 19301 during a DSC meeting. 
https://github.com/department-of-veterans-affairs/va.gov-team/issues/19301

**Comment by caw310** [comment]
@GnatalieH , no need to do anything on this. Just prepping as this is on the list of priorities from DSC.

**Comment by GnatalieH** [comment]
There are no more examples I've found other than the VAOS appointment cards that incorrectly use bold or strong text where a heading should be used. I spoke with Noah, Angela and Josh, and they also couldn't think of any other examples. Josh said that they tend to call out misuse of bold or strong text right away and those issues get fixed quickly. We also decided that doing a search of `strong` (if that could be done) in the code would result in too many false positives to be useful. I'll be working on documenting the proper use of headings with @RLHecht in early 2022.
